appdatabase/migrations/sql/1721832718_rename_shard_test.up.sql

```diff
@@ -0,0 +1,28 @@
+UPDATE settings SET fleet = "status.prod" WHERE fleet = "shards.test";
+
+UPDATE cluster_config SET fleet = "status.prod" WHERE fleet = "shards.test";
+
+UPDATE cluster_nodes
+SET node = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.prod.status.nodes.status.im"
+WHERE node = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.test.shards.nodes.status.im"
+AND type = "discV5boot";
+
+UPDATE cluster_nodes
+SET node = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.prod.status.nodes.status.im"
+WHERE node = "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.test.shards.nodes.status.im"
+AND type = "waku";
+
+UPDATE cluster_nodes
+SET node = "enr:-QEKuED9AJm2HGgrRpVaJY2nj68ao_QiPeUT43sK-aRM7sMJ6R4G11OSDOwnvVacgN1sTw-K7soC5dzHDFZgZkHU0u-XAYJpZIJ2NIJpcISnYxMvim11bHRpYWRkcnO4WgAqNiVib290LTAxLmRvLWFtczMuc3RhdHVzLnByb2Quc3RhdHVzLmltBnZfACw2JWJvb3QtMDEuZG8tYW1zMy5zdGF0dXMucHJvZC5zdGF0dXMuaW0GAbveA4Jyc40AEAUAAQAgAEAAgAEAiXNlY3AyNTZrMaEC3rRtFQSgc24uWewzXaxTY8hDAHB8sgnxr9k8Rjb5GeSDdGNwgnZfg3VkcIIjKIV3YWt1Mg0"
```

Comment:
The enr listed here can be retrieved through `enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.prod.status.nodes.status.im`, right? If yes, wondering why we duplicate these enr records.

Reply:
If for some reason dns discovery fails to retrieve the results (i.e.: domain is blocked), the app will still attempt to connect to those nodes directly since their ENR use IP Address.